components/authz-service/server/v2/project_test.go

```diff
@@ -371,9 +371,14 @@ func addProjectToStore(t *testing.T, store *cache.Cache, id, name string, projTy
 }
 
 func setupProjects(t *testing.T) (api.ProjectsClient, *cache.Cache, *mockEventServiceClient) {
+	cl, ca, _, mc, _ := setupProjectsAndRules(t)
+	return cl, ca, mc
+}
+func setupProjectsAndRules(t *testing.T) (api.ProjectsClient, *cache.Cache, *cache.Cache, *mockEventServiceClient,
+	int64) {
 	t.Helper()
 	ctx := context.Background()
-	prng.Seed(t)
+	seed := prng.GenSeed(t)
```

Comment:
Why the seeding change?

Reply:
because it's controlling randomness in gopter